web/firefoxcom.js

```diff
@@ -294,6 +294,10 @@ class FirefoxScripting {
     FirefoxCom.request("dispatchEventInSandbox", event);
   }
 
+  static async dispatchAsyncEventInSandbox(event) {
+    return FirefoxCom.requestAsync("dispatchAsyncEventInSandbox", event);
+  }
+
```

Comment:
Can we please just modify the method above as follows instead (and similarly in the `web/generic_scripting.js` file).
```javascript
static async dispatchEventInSandbox(event, awaitResponse = false) {
  if (awaitResponse) {
    return FirefoxCom.requestAsync("dispatchAsyncEventInSandbox", event);
  }
  return FirefoxCom.request("dispatchEventInSandbox", event);
}
```

(Since I somewhat doubt that the run-time overhead will even be measurable, and adding essentially duplicated code never feels great)